source/index.html.md

```diff
@@ -1195,6 +1195,7 @@ Parameter | Type | Description
 `phrases` (optional) | [string] | A list of strings containing words and phrases "hints" so that the audio transcription is more likely to recognize them. This can be used to improve the accuracy for specific words and phrases, or to add additional words to the vocabulary for the transcription.
 `urgency` (optional, default `day`) | string | A string describing the urgency of the response. One of `immediate`, `day`, or `week`, where `immediate` is a one-hour response time.
 `metadata` (optional, default `{}`) | object | A set of key/value pairs that you can attach to a task object. It can be useful for storing additional information about the task in a structured format.
+`skip_human_transcription` (optional, default `false`) | boolean | Specifies whether or not to skip human transcription.
```

Comment:
Makes sense! It'd be reasonable to mention it in the text portion of the docs too if the table is getting too crowded (though some of the other params in the table are already pretty long)